ceph.spec.in

```diff
@@ -420,6 +420,9 @@ Base is the package that includes all the files shared amongst ceph servers
 %package -n cephadm
 Summary:        Utility to bootstrap Ceph clusters
 Requires:       lvm2
+%if 0%{?suse_version}
+Requires:       apparmor-profiles
```

Comment:
This should be enough from my point of view:
```suggestion
Requires:       apparmor-abstractions
```

Reply:
updated!